integration-tests/js-compute/fixtures/backend/backend.js

```diff
@@ -113,6 +113,13 @@ function handleRequest(event) {
     return new Response("Compute SDK Test Backend");
   }
 
+  // tee
+
+  if (method == "POST" && url.pathname == "/tee") {
+    const text = await event.request.text();
+    return new Response(text);
+  }
```

Comment:
Adding a new testing endpoint that can return a post body for use with the `tee` test.